make-release.sh

```diff
@@ -291,7 +291,7 @@ buildImages() {
     fi
 
     # BUILD IMAGES
-    bash "$(pwd)/build/build.sh" --tag:${TAG}
+    bash "$(pwd)/che-server/build/build.sh" --tag:${TAG}
```

Comment:
This looks like it should work, though we might be safer taking the directory relative to the script's directory rather than `pwd` -- the latter can break again if `./che-server/make-release.sh` in the release action changes. 

I usually use something like
```bash
SCRIPT_DIR=$(cd "$(dirname "$0")" || exit; pwd)
```
to get the absolute path of the current script on disk, then take relative paths from that directory (i.e. `${SCRIPT_DIR}/build/build.sh`